applications/GeoMechanicsApplication/tests/1d_consolidation.py

```diff
@@ -0,0 +1,107 @@
+        file_path = test_helper.get_file_path(os.path.join('.', test_name))
+        parameter_file_names = [os.path.join(file_path, 'ProjectParameters_stage' + str(i + 1) + '.json') for i in
+                                range(n_stages)]
+
+        # set stage parameters
+        parameters_stages = [None] * n_stages
+        os.chdir(file_path)
+        for idx, parameter_file_name in enumerate(parameter_file_names):
+            with open(parameter_file_name, 'r') as parameter_file:
+                parameters_stages[idx] = Kratos.Parameters(parameter_file.read())
+
+        model = Kratos.Model()
+        stages = [analysis.GeoMechanicsAnalysis(model, stage_parameters) for stage_parameters in parameters_stages]
+
+        # run stages and get water pressure/displacement results per stage
+        stage_water_pressure = [None] * n_stages
+        stage_displacement   = [None] * n_stages
+        for idx, stage in enumerate(stages):
+            stage.Run()
+            stage_water_pressure[idx] = test_helper.get_water_pressure(stage)
+            displacements = test_helper.get_displacement(stage)
+            stage_displacement[idx]   = [displacement[1] for displacement in displacements] 
+
+        # get y coords of all the nodes
+        coords = test_helper.get_nodal_coordinates(stages[0])
+        y_coords = [coord[1] + 1 for coord in coords]
+        
+        t_vs = [0.01, 0.02, 0.05, 0.1, 0.2, 0.5, 1.0, 2.0, 5.0, 10.0]
+        # calculate water pressure analytical solution for all stages and calculate the error
+        sample_height = 1
+        rmse_stages = [None] * (n_stages - 1)
+        for idx, t_v in enumerate(t_vs):
+            rel_p_stage  = [calculate_pore_pressure_1D_consolidation(y_coord, sample_height, t_v) * -1 for y_coord in y_coords]
+            errors_stage = [stage_water_pressure[idx + 1][node_idx] - rel_p for node_idx, rel_p in
+                            enumerate(rel_p_stage)]
+            rmse_stages[idx] = (sum([error ** 2 for error in errors_stage]) / len(errors_stage)) ** 0.5
+
+        # assert if average error in all stages is below 1 percent
+        accuracy = 0.01
+        for rmse_stage in rmse_stages:
+            self.assertLess(rmse_stage, accuracy)
+
+        # calculate the degree of consolidation analytical solution for all stages and calculate the error
+        # Veruijt's notations
+        q = -1.0
+        K = 3.33e2
+        G = 5.0e2
+        m_v = 1/(K + 4/3 * G)
+        delta_h8 = (-1)*m_v*sample_height*q
+        beta = 0.5e-9
+        n = 0.3
+        delta_h0 = (-1)*m_v*sample_height*q*n*beta/(m_v+n*beta)
+        for idx, t_v in enumerate(t_vs):
+            rel_displacement = [(stage_displacement[idx + 1][node_idx]-delta_h0)/(delta_h8-delta_h0) for node_idx, y_coord in
+                            enumerate(y_coords) if fabs(y_coord - sample_height) < 0.001]
+            degree_of_consolidation = [calculate_degree_of_1D_consolidation(sample_height, t_v) * -1 
```

Comment:
Maybe we should add to the variable name that this is the analytical degree of consolidation
```suggestion
            analytical_degree_of_consolidation = [calculate_degree_of_1D_consolidation(sample_height, t_v) * -1 
```

Reply:
Agreed. Changed.